resources/building-mobile-extensions/requesting-a-shared-state.md

```diff
@@ -4,17 +4,8 @@ Extensions request shared states by using the `ACPExtensionApi` \(iOS\) / `Exten
 
 The example below shows a typical scenario where shared state is requested. In this scenario, a previously registered event listener is called, and the implementation gets the latest configuration shared state passing the current event as context.
```

Comment:
please change "shared state passing the..." to "shared state that passes the..."

Reply:
Here it makes more sense to say `gets the latest configuration shared state by passing the current event as context`